shared/oae/css/oae.base.css

```diff
@@ -464,14 +464,24 @@ ul.as-selections li.as-original input::-ms-clear {
 }
 
 ul.as-selections li.as-selection-item {
-    border-radius: 3px;
-    box-shadow: none;
+    -webkit-border-radius: 3px;
+       -moz-border-radius: 3px;
+            border-radius: 3px;
+    -webkit-box-shadow: none;
```

Comment:
I don't think we need to do vendor prefixes for `box-shadow` either. You can remove these + any other `box-shadow` vendor prefixes we have